idea-plugin/src/main/java/com/palantir/gradle/jdks/GradleJdksProjectService.java

```diff
@@ -90,7 +110,28 @@ public void maybeSetupGradleJdks() {
                     "Skipping setupGradleJdks because gradle JDK setup is not found %s", gradleSetupScript));
             return;
         }
-        setupGradleJdks();
+        TasksKt.withBackgroundProgress(
```

Comment:
instead of: https://github.com/JetBrains/intellij-community/blob/99964dd3c76f6cf9eb24018c002a68848e54dddb/platform/core-api/src/com/intellij/openapi/progress/Task.java#L43C8-L43C12 which has obsolescence notice